src/movie/List.js

```diff
@@ -28,6 +28,9 @@ const mapDispatchToProps = {
 class MovieList extends Component {
   constructor(props) {
     super(props);
+    if (props.query) {
+      this.handleQueryChange();
+    }
```

Comment:
why did you write it not in componentDidMount ??

Reply:
Because we need to display the result as per search query on load itself. If I move it to `componentDidMount` then first it will render all records and then  again it would re-render as per query.